android/src/main/java/com/dylanvann/fastimage/FastImageViewModule.java

```diff
@@ -45,12 +56,47 @@ public void run() {
                             //    - data:image/png;base64
                             .load(
                                     imageSource.isBase64Resource() ? imageSource.getSource() :
-                                    imageSource.isResource() ? imageSource.getUri() : imageSource.getGlideUrl()
+                                            imageSource.isResource() ? imageSource.getUri() : imageSource.getGlideUrl()
                             )
+
                             .apply(FastImageViewConverter.getOptions(source))
                             .preload();
                 }
             }
         });
     }
+
+    @ReactMethod
+    public void loadImage(final ReadableMap source, final Promise promise) {
+        final Activity activity = getCurrentActivity();
+        if (activity == null) return;
+        activity.runOnUiThread(new Runnable() {
+            @Override
+            public void run() {
+                final FastImageSource imageSource = FastImageViewConverter.getImageSource(activity, source);
+
```

Comment:
And finally that empty line, as it seems there is no empty line between variable declaration in the source code ^^